docs/userguide.md

```diff
@@ -43,8 +43,10 @@ custom image](development.md#test-your-changes).
 
 ### Create Clusters
 
-The quickest way to set up clusters for use with the federation-v2 control
-plane is to use [minikube](https://kubernetes.io/docs/getting-started-guides/minikube/).
+The federation v2 control plane can run on any Kubernetes clusters >= v1.11. Steps for running federation v2 on
+Google Kubernetes Engine [(GKE)](https://cloud.google.com/kubernetes-engine/) have been [documented](./gke.md). However,
```

Comment:
Please create a new `Cloud Provider` setup subsection within this `Create Clusters` section and start a new list of providers with a pointer to the `GKE` doc. That way we can grow the list without rewriting text.